sourced/ml/__main__.py

```diff
@@ -57,6 +57,28 @@ def add_parser(name, help):
     add_engine_args(repos2df_parser)
     add_feature_args(repos2df_parser)
     # ------------------------------------------------------------------------
+    repos2ids_parser = subparsers.add_parser(
+        "repos2ids", help="Convert source code to a bag of identifiers.")
+    repos2ids_parser.set_defaults(handler=repos2ids_entry)
+    add_engine_args(repos2ids_parser)
+    repos2ids_parser.add_argument(
+        "-r", "--repositories", required=True,
+        help="The path to the repositories.")
+    repos2ids_parser.add_argument(
+        "-o", "--output", required=True,
+        help="[OUT] The path to the output csv file of identifiers, compressed in gz format.")
+    repos2ids_parser.add_argument(
+        "--split", action="store_true",
+        help="Flag that specifies if you want to filter specific identifiers."
```

Comment:
Make it shorter:
Be shorter:
```
Enables filtering identifiers...
```

Honestly, I don't get the help message at all. What do you mean?